hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/api/db.json

```diff
@@ -684,9 +684,9 @@
         "status": "OPEN",
         "leaderNode": "localhost1.storage.enterprise.com",
         "datanodes": [
-          "localhost1.storage.enterprise.com",
-          "localhost2.storage.enterprise.com",
-          "localhost3.storage.enterprise.com"
+          "localhost1.storage.enterprise.com/7d8c6c89-6bd2-43cd-974d-a54344eb46bf",
```

Comment:
I think it's better to extend API and make "datanodes" items an object like DatanodeDetails instead of just a string.

Reply:
the patch was updated by @smitajoshi12, could you take a look at it @myskov?